pyproject.toml

```diff
@@ -3,6 +3,19 @@ name = "shared-memory-dict"
 version = "0.1.0"
 description = "A very simple shared memory dict implementation"
 authors = ["Arquitetura LuizaLabs <arquitetura@luizalabs.com>"]
+license = "MIT"
+readme = "README.md"
+homepage = "https://github.com/luizalabs/shared-memory-dict"
+repository = "https://github.com/luizalabs/shared-memory-dict"
+keywords = ["django", "aiocache", "shared memory"]
+classifiers = [
+    "Operating System :: OS Independent",
+    "License :: OSI Approved :: MIT License",
+    "Topic :: Software Development :: Libraries :: Python Modules",
```

Comment:
What about including the ` 'Programming Language :: Python :: 3.8',` classifier here?

Reply:
is It really necessary?
Because poetry does add it on `classifiers` [take a look](https://test.pypi.org/project/shared-memory-dict/)